site/src/pages/release-notes/1.28.0.mdx

```diff
@@ -0,0 +1,199 @@
+---
+date: 2024-04-12
+---
+
+## 🌟 New features
+
+- You can now create a Spring HTTP Interface backed by Armeria's <type://WebClient>. #5370
+```java
+WebClient webClient = ...;
```

Comment:
We need to indent the code block so that the block is part of the list item:

```
- foo
  '''java
  block
  '''
  - bar
    '''java
    block 2
    '''
```

Reply:
I see, done!